code/modules/clothing/clothing.dm

```diff
@@ -719,6 +725,8 @@ BLIND     // can't see anything
 	..()
 
 /obj/item/clothing/under/proc/attach_accessory(obj/item/clothing/accessory/A, mob/user, unequip = FALSE)
+	if(!istype(A))
+		return FALSE
 	if(can_attach_accessory(A))
```

Comment:
remove this safety check. if a none accessory is geting passed into this proc we want it to runtime so we can find the issue.

Reply:
done